CHANGELOG.md

```diff
@@ -34,6 +34,10 @@ To learn more about our roadmap, we recommend reading [this document](ROADMAP.md
 
 ## Unreleased
 
+### Improvements
+
+- **AWS SQS Queue:** Updated the sqs scaler to add the config flag `scaleOnInFlight`. ([#773](https://github.com/kedacore/keda-docs/pull/773))
```

Comment:
```suggestion
- **AWS SQS Queue:** Support for scaling to include in-flight messages. ([#3133](https://github.com/kedacore/keda/issues/3133))
```

Reply:
I have updated the wording, links and moved it to the correct location 👍🏻